package.json

```diff
@@ -110,7 +110,7 @@
     "typescript": "2.1.5",
     "wallaby-webpack": "0.0.30",
     "webdriver-manager": "11.1.1",
-    "zone.js": "0.7.5"
```

Comment:
I upgraded zone.js because at first I was getting unclear errors because of this issue https://github.com/angular/angular-cli/issues/3975

After updating the reason became apparent but I could roll this back if needed